specifyweb/stored_queries/build_models.py

```diff
@@ -78,11 +76,11 @@ def make_relationship(reldef):
             column = getattr(table.c, reldef.column)
 
             relationship_args = {'foreign_keys': column}
+            if remote_class is cls:
+                relationship_args['remote_side'] = table.c[ tabledef.idColumn ]
 
             if hasattr(reldef, 'otherSideName'):
                 backref_args = {'uselist': reldef.type != 'one-to-one'}
-                if remote_class is cls:
-                    backref_args['remote_side'] = table.c[ tabledef.idColumn ]
 
                 relationship_args['backref'] = orm.backref(reldef.otherSideName, **backref_args)
```

Comment:
We will have to look at refactoring this code in the future. 
sqlalchemy has "deprecated" `orm.backref` in favor of `relationship.back_populates` explicitly using `relationship` objects. 
https://docs.sqlalchemy.org/en/20/orm/backref.html

I'm sure there are plenty of other things like this as well that we should address.

Reply:
Ah, yes, no worries then. 
This works for now.